examples/squeezer_bin/main.rs

```diff
@@ -14,7 +14,9 @@ fn main() -> glib::ExitCode {
         let mode_switch = gtk::Switch::new();
         let switch_label = gtk::Label::new(Some("keep aspect ratio"));
         let squeezer = SqueezerBin::default();
-        squeezer.set_child(Some(&gtk::Label::new(Some("Hello World!"))));
+        squeezer.set_child(Some(
+            gtk::Label::new(Some("Hello World!")).upcast::<gtk::Widget>(),
```

Comment:
Can you explain the reason for adding `.upcast::<gtk::Widget>()`?

When I remove this, the program runs without errors and behaves as I as expected.

Reply:
Does it? Because the setter expects a GtkWidget and we can't have a generic parameter. If it works, then send a patch ;)

Reply:
It seems to work without the upcast, so I opened #1763